### DISCUSSION TITLE
Large image push fails..

### DISCUSSION THREAD

**Discussion by prashi2202** [goharbor/harbor#15529] [Q&A]
Hi,

Am trying to push a 70GB image into the harbor repo which is connected to S3 Bucket is failing with following error message..

Aug 31 07:40:20 vas-006 dockerd[99851]: time="2021-08-31T07:40:20.249091700Z" level=error msg="Upload failed: blob upload unknown"
Aug 31 07:40:20 vas-006 dockerd[99851]: time="2021-08-31T07:40:20.249200905Z" level=info msg="Attempting next endpoint for push after error: blob upload unknown"

Following are the nginx annotations:

      nginx.ingress.kubernetes.io/ssl-redirect: "true"
      nginx.ingress.kubernetes.io/proxy-body-size: "0"
      ingress.kubernetes.io/proxy-body-size: "0"
      nginx.ingress.kubernetes.io/proxy-buffering: "off"
      nginx.ingress.kubernetes.io/proxy-request-buffering: "off"

I was also going through few articles and came across the below git link that talks about large image push failures and the solution was to include "proxy_set_header X-Forwarded-Proto https" ...

https://github.com/distribution/distribution/issues/2225 

Is this applicable to harbor as well, can someone help with a solution to this issue?

**Comment by wy65701436** [comment]
did you try to proxy_set_header X-Forwarded-Proto https? does it work?
